lamindb/curators/core.py

```diff
@@ -20,7 +20,7 @@
 import lamindb_setup as ln_setup
 import numpy as np
 import pandas as pd
-import pandera
+import pandera.pandas as pa
```

Comment:
As I pointed out in the other PR, AFAIK, `pa` is typically used for `pyarrow`. We'll merge as is now, but I think we should use something that is less confusing. If `pandera` doesn't work anymore maybe `ppd` or something like this.